lib/active_model/serializer/adapter/json.rb

```diff
@@ -37,9 +37,7 @@ def serializable_hash(options = {})
             @result = @core.merge @hash
           end
 
-          if root = options.fetch(:root, serializer.json_key)
-            @result = { root => @result }
-          end
+          @result = { root => @result } if root
           @result
```

Comment:
I think it could be debated for either way :smile: but it's always interesting to hear other people's opinions on coding style. Happy to change it if others think it needed.